sound/soc/intel/boards/Makefile

```diff
@@ -53,7 +53,7 @@ obj-$(CONFIG_SND_SOC_INTEL_BXT_RT298_MACH) += snd-soc-sst-bxt-rt298.o
 obj-$(CONFIG_SND_SOC_INTEL_SOF_PCM512x_MACH) += snd-soc-sst-sof-pcm512x.o
 obj-$(CONFIG_SND_SOC_INTEL_SOF_WM8804_MACH) += snd-soc-sst-sof-wm8804.o
 obj-$(CONFIG_SND_SOC_INTEL_GLK_RT5682_MAX98357A_MACH) += snd-soc-sst-glk-rt5682_max98357a.o
-obj-$(CONFIG_SND_SOC_INTEL_BROADWELL_MACH) += snd-soc-sst-broadwell.o
+obj-$(CONFIG_SND_SOC_INTEL_BROADWELL_MACH) += snd-soc-bdw-rt286.o
```

Comment:
we've already discussed this, this will break our CI scripts

if you do this rename, you also need to add the new name to this file.
https://github.com/thesofproject/sof-test/blob/main/tools/kmod/sof_remove.sh#L142

Reply:
Forgot about this. Will do and link it here.

Reply:
Opened new [PR](https://github.com/thesofproject/sof-test/pull/949) in sof-test to address the request.